node/flows.re

```diff
@@ -411,3 +411,16 @@ let request_withdraw_proof = (state, ~hash) =>
   };
 let request_ticket_balance = (state, ~ticket, ~address) =>
   state.Node.protocol.ledger |> Ledger.balance(address, ticket);
+
+let trusted_validators_membership = (state, update_state, request) => {
```

Comment:
This operation needs to be signed by the validator key, otherwise anyone could register a key as trusted validator

Reply:
Addressed in the incoming commit